packages/components/tabs/src/tabs.test.ts

```diff
@@ -0,0 +1,53 @@
+import { describe, expect, it } from 'vitest'
+import { mount } from '@vue/test-utils'
+import { nextTick } from 'vue'
+import TabsDemo from './stories/TabsDemo.vue'
```

Comment:
@ejirocodes you need to write tests for the actual tabs components and not story comp